### PR TITLE
Easier support for multiple instances locally 

### PR DESCRIPTION
**Pull request by jakobht**
**What changed?**
- Make ports in development config configurable, so we can start several instances of the same service.
    - We keep the old defaults, so if nothing is specified everything works as before
- Added script that starts several instances of the services, mainly for illustrative purposes, but could be useful to quickly start 3 instances of each of frontend, history and matching.

**Why?**
- As we want to test things related to sharing we need to be able to have multiple instances of the services running, this enables this. 

**How did you test it?**
Ran locally, this is just a development setup

**Potential risks**
None

**Release notes**

**Documentation Changes**

### CODE REVIEW

File: scripts/run_several_instances.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+MATCHING_PORT=11002 MATCHING_PORT_GRPC=11102 MATCHING_PORT_PPROF=11202 \
+  ./cadence-server --env development start --services matching &
+
+# Start two more instances of the history service on different ports
+HISTORY_PORT=12001 HISTORY_PORT_GRPC=12101 HISTORY_PORT_PPROF=12201 \
+  ./cadence-server --env development start --services history &
+HISTORY_PORT=12002 HISTORY_PORT_GRPC=12102 HISTORY_PORT_PPROF=12202 \
+  ./cadence-server --env development start --services history &
+
+wait
```
Comment (taylanisikdemir) [inline scripts/run_several_instances.sh:29]: you need to add all these new ports to `ringpop.bootstrapHosts` in the development.yaml

**Review by taylanisikdemir — APPROVED** [review]

File: scripts/run_several_instances.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```
Comment (taylanisikdemir) [inline scripts/run_several_instances.sh:1]: a script to stop all these binaries would be nice too

File: scripts/run_several_instances.sh (hunk 1 above)
Comment (taylanisikdemir) [reply scripts/run_several_instances.sh:29]: discussed offline. this is optional. the individual services should find each other thru one of the hosts specified in bootstrap hosts. Jakob confirmed it's working as expected.

File: scripts/run_several_instances.sh (hunk 2 above)
Comment (jakobht) [reply scripts/run_several_instances.sh:1]: Modified the script so ctrl-c will kill all the services. So now we can run the script, and if we just want to kill everything we just do ctrl-c and everything will be instantly killed.

**Review by taylanisikdemir — APPROVED** [review]

**Pull request merged**